lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -60,6 +60,11 @@ class Chef::Knife::VsphereVmSnapshot < Chef::Knife::BaseVsphereCommand
          description: 'Quiesce the VM prior to snapshotting',
          default: false
 
+  option :snapshot_description,
+         long: '--snapshot-descr DESCR',
+         description: 'Snapshot description',
+         default: ' '
```

Comment:
Could you switch this to be an empty string like we used to set, please?

Reply:
um, I'm not sure what you mean?  I need to allow for a snapshot description to be entered so the description can be stored against the created snapshot.  At the moment, an empty string is hardcoded for the description so we can't enter a description at all.  Is this what you mean?  I'm a little confused??

Reply:
Hi, sorry for not being more clear. Previously it would use `''` and the PR makes the default description a space (`' '`). I'm asking if you could drop the space and just use `''` as a default.

Reply:
hehehe....oh yes, I understand you now....yep, I can change that...I'll do it tonight when I get home....thanks :)